### DISCUSSION TITLE
Links to our Lab Manual Pages!

### DISCUSSION THREAD

**Discussion by shannonrankin** [nmfs-ost/PAM-OpenScience#7] [General]
Reply to this discussion with a link to your Lab Manual pages (website)

**Comment by Kourtney-Burger** [comment]
[SWFSC SAEL](https://sael-swfsc.github.io/SAEL-lab-manual/)

**Comment by cberchok** [comment]
https://noaa-afsc.github.io/AFSC-MML-PAM-Lab-Manual/

**Comment by rebecca-vanhoeck** [comment]
NEFSC Lab Manual Link: [NEFSC PAB](https://nefsc.github.io/NEFSC_PAB_lab_manual/)

**Reply by Kourtney-Burger** [reply]
Hi Becca, 

Can you update this link? Thanks!

**Reply by rebecca-vanhoeck** [reply]
Thanks for the reminder, Kourtney!

**Comment by emmonsca** [comment]
https://noaa-nwfsc.github.io/NWFSC-PAM-Lab-Manual/

**Comment by HFrouinMouy** [comment]
Test (practice) [Heloise](https://hfrouinmouy.github.io/SEFSC-PAM-Lab-Manual/)

**Comment by lynne-hodge** [comment]
https://github.com/SEFSC/SEFSC-PAM-Lab-Manual

**Comment by jlkeating** [comment]
https://pifsc-protected-species-division.github.io/CRP_PAM_Manual/
